modules/runtime/nuxeo-runtime/src/main/java/org/nuxeo/runtime/model/impl/RegistrationInfoImpl.java

```diff
@@ -72,8 +72,12 @@
     @XNode("@name")
     ComponentName name;
 
-    @XNode("@disabled")
-    boolean disabled;
+    @XNode("@disable")
```

Comment:
We're sure we don't need compat with the old name?

Reply:
Disablement API was only used in unused persisted registrations so I think we do not need compat indeed.

Reply:
Why not removing it and leverage `enable`?

Reply:
The idea is to be able to avoid scripting logics, so it's easier to have:
```
<component enable="${org.nuxeo.my.condition}">
<component>
```
for components to *enable* conditionally if org.nuxeo.my.condition runtime property is true, and also:
```
<component disable="${org.nuxeo.my.condition}">
<component>
```
for components to *disable* conditionally (on the same condition).

The alternative would be to introduce some scripting logics like

```
<component enable="!${org.nuxeo.my.condition}">
</component>
```

Reply:
Ok, it's clearer, thanks 👍.

I think the scripting logic is the way to go, otherwise, I'm pretty sure we're gonna end with:
```xml
<component enable="${org.nuxeo.my.condition.1}" disable="${org.nuxeo.my.condition.2}">
</component>
```
I'm afraid about maintenance on this.

Furthermore, improving the scripting logic could be interesting as we may need to have OR and AND boolean operators in the future. 
This may be needed by drive change finder which has 3 implementations and only one might be enabled.

Reply:
Well if we need scripting logics, i'm afraid using nxftl is a better option: the goal of this feature was to avoid having scripting logics introduced for simple conditional enablements and disablements, and maybe simplify some configuration templates using (or over-using) freemarker templating for components configuration.
This does not prevent from keeping nxftl, and i do not see the point of this PR anymore if we're not happy with this improvement: i've discussed this several times with you both and you seemed in favor of it so i'm confused (but it's ok if you changed your minds, this was a quick try at simplifying some use cases).